src/VisualStudio/Core/Def/Implementation/Remote/GlobalNotificationRemoteDeliveryService.cs

```diff
@@ -76,7 +76,7 @@ private void OnGlobalOperationStarted(object sender, EventArgs e)
             lock (_globalNotificationsGate)
             {
                 _globalNotificationsTask = _globalNotificationsTask.SafeContinueWithFromAsync(
-                    SendStartNotificationAsync, _cancellationToken, TaskContinuationOptions.None, TaskScheduler.Default);
+                    SendStartNotificationAsync, _cancellationToken, TaskContinuationOptions.OnlyOnRanToCompletion, TaskScheduler.Default);
```

Comment:
⚠️ This may cause `_globalNotificationsTask` to _latch_ in a cancelled state. Is this the behavior we want?

Reply:
Not sure what you mean.

Reply:
Do you mean that we stop processing the tasks once one of them fails/cancels? I think that's ok. These are not supposed to be canceled unless VS is shutting down. They might fail if OOP fails, after which there is not point in sending more notifications to OOP.

Reply:
I agree with this interpretation

Reply:
This makes sense, although I would love it if we put a comment to that effect in the code.